google/cloud/bigtable/data/_async/client.py

```diff
@@ -566,13 +609,16 @@ async def read_rows(
         *,
         operation_timeout: float | TABLE_DEFAULT = TABLE_DEFAULT.READ_ROWS,
         attempt_timeout: float | None | TABLE_DEFAULT = TABLE_DEFAULT.READ_ROWS,
+        retryable_errors: Sequence[grpc.StatusCode | int | type[Exception]]
+        | TABLE_DEFAULT = TABLE_DEFAULT.READ_ROWS,
```

Comment:
Should TABLE_DEFAULT enum be prefixed with an underscore? My understanding is that its used as a marker value to signal that the user passed something in. I dont think it makes sense for endusers to be able to set it

Reply:
I can see arguments either way. TABLE_DEFAULT already exists in the codebase for handling timeouts though, so if we want to change it, maybe it should be a separate PR?